sickbeard/helpers.py

```diff
@@ -1561,3 +1561,18 @@ def canonical_name(obj, fmt='{key}:{value}', separator='|', ignore_list=frozense
     """
     guess = obj if isinstance(obj, dict) else guessit.guessit(obj)
     return str(separator.join([fmt.format(key=k, value=v) for k, v in guess.items() if k not in ignore_list]))
+
+
+def get_broken_providers():
+    """Get broken providers from cdn.pymedusa.com."""
+    url = 'https://cdn.pymedusa.com/providers/broken_providers.txt'
```

Comment:
@fernandog is there a need for that since if they can't connect to the cdn they probably don't have internet so they can't connect to the providers anyway?